tdvt/tdvt/config_gen/tdvtconfig.py

```diff
@@ -4,33 +4,43 @@
 import re
 
 from ..resources import *
+from .test_config import TestConfig, RunTimeTestConfig
 
-class TdvtTestConfig(object):
+class TdvtInvocation(object):
     """Track how items were tested. This captures how tdvt was invoked."""
-    def __init__(self, tested_sql=False, tested_tuples=True, tds='', config='', output_dir='', logical=False, verbose=False, override='', suite_name='', from_args=None, thread_count=6, from_json=None, run_as_perf=False, timeout_seconds=60 * 60):
-        self.tested_sql = tested_sql
-        self.tested_tuples = tested_tuples
+    def __init__(self, from_args=None, from_json=None, test_config : TestConfig = None):
```

Comment:
```suggestion
    def __init__(self, from_args=None, from_json=None, test_config: TestConfig = None):
```
I think the spacing matters for type hints to be recognized